### DISCUSSION TITLE
Do manifest files support dependencies on other winget packages?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/winget-pkgs#18942] [Ideas] [answered]
For example I want to get dependency python before my program gets installed. Are dependencies supported for such operations?

**Comment by OfficialEsco** [comment, marked as answer]
Yes and no, the [Schema](https://aka.ms/winget-manifest.installer.1.0.0.schema.json) does support it, but the function is not added yet https://github.com/microsoft/winget-cli/issues/163.
You can [add them](https://github.com/microsoft/winget-pkgs/blob/4db2245198df3f7b0cdfbe2a00a79f60783e3e9b/manifests/e/ElectronicArts/EADesktop/12.0.100.4941/ElectronicArts.EADesktop.installer.yaml) but they don't actually do anything as of now.
